### ISSUE TITLE
Why is there a significant difference between the predicted results and the actual results

### ISSUE THREAD

**Issue by pandasayz** [lululxvi/deepxde#448]
Can someone help me? Does anyone know why the predicted result I got is not consistent with the real result? Below is my drawing. The yellow line is the real value, and the blue is the predicted value calculated by the trained model：
![t10_5-4_15000](https://user-images.githubusercontent.com/96182400/146294314-9f68539c-3a9f-432f-b78c-732217688930.png)

**Comment by pandasayz** [comment]
The network structure is [2] + [50] * 4 + [1]，and Network trained 15000  times

**Comment by xuliang5115** [comment]
Please feel free to show your code include PDE and Boundary condition...

**Comment by pandasayz** [comment]
thank you for your attention.Here are my procedures and equations：
![ac](https://user-images.githubusercontent.com/96182400/146299030-04490691-6b23-4ed6-9b71-40601373eb59.png)
`import deepxde as dde
import numpy as np
import scipy.io
import matplotlib.pyplot as plt


def gen_testdata():
    data = scipy.io.loadmat(r'E:\python_ex\data\AC.mat')

    t = data['tt'].flatten()[:, None]
    x = data['x'].flatten()[:, None]
    exact = data['uu']
    xx, tt = np.meshgrid(x, t)
    X = np.vstack((np.ravel(xx), np.ravel(tt))).T
    y = exact.flatten()[:, None]
    return X, y


def pde(x, y):
    dy_t = dde.grad.jacobian(y, x, i=0, j=1)
    dy_xx = dde.grad.hessian(y, x, i=0, j=0)
    return dy_t - 0.0001 * dy_xx + 5 * y ** 3 - 5 * y


def boundary(x, on_boundary):
    return on_boundary and np.isclose(x[0], -1)


geom = dde.geometry.Interval(-1, 1)
timedomain = dde.geometry.TimeDomain(0, 1)
geomtime = dde.geometry.GeometryXTime(geom, timedomain)

bc1 = dde.PeriodicBC(geomtime, 1, boundary, derivative_order=0)
bc2 = dde.PeriodicBC(geomtime, 1, boundary, derivative_order=1)
ic = dde.IC(
    geomtime, lambda x: x[:, 0:1] ** 2 * np.cos(np.pi * x[:, 0:1]), lambda _, on_initial: on_initial
)

data = dde.data.TimePDE(
    geomtime, pde, [bc1, bc2, ic], num_domain=2540, num_boundary=80, num_initial=100
)
net = dde.maps.FNN([2] + [50] * 4 + [1], "tanh", "Glorot normal")
model = dde.Model(data, net)

model.compile("adam", lr=1e-3)
model.train(epochs=15000)
model.compile("L-BFGS")
losshistory, train_state = model.train()
dde.saveplot(losshistory, train_state, issave=True, isplot=True)`

**Comment by pandasayz** [comment]
> 
The predictions above are comparable t=10，and the prediction result at t=90  is even worse

**Comment by xuliang5115** [comment]
There is no loss history so we can not confirm the problems.
May be you need add loss weights or scaling.

loss weights like this:
```
loss_weights = [1, 1, 100, 100] # change this 
model.compile("adam", lr=0.01,
              loss_weights=loss_weights)
```

**Comment by pandasayz** [comment]
> There is no loss history so we can not confirm the problems. May be you need add loss weights or scaling.
> 
> loss weights like this:
> 
> ```
> loss_weights = [1, 1, 100, 100] # change this 
> model.compile("adam", lr=0.01,
>               loss_weights=loss_weights)
> ```

Thank you very much for your answer.it seems work well.I haven't seen such a solution in the article(deepxde). Could you please tell me where I can see the theory? I want to know the principle and how to adjust it.

**Comment by pandasayz** [comment]
![t10](https://user-images.githubusercontent.com/96182400/146307331-fd742e8c-54b6-491c-86aa-2a37773080ed.png)
This is the result after adjustment. Thank you again！！！

**Comment by pandasayz** [comment]
![t90](https://user-images.githubusercontent.com/96182400/146309352-41a21e5a-7ba2-4b17-be2f-b7395301f332.png)
the result at t=0.9 is still poor ,Can you tell me which adjustment I should refer to？

**Comment by xuliang5115** [comment]
Because multi-object penalty coefficient are unbalanced, so we need adjust it, Wang use (NTK or GP) adapty weights method, Deepxde set by parameter loss weights.
Model poor performs has multi reason,so we can't make true which important for the problem even this is only a simple problem. The best way is add model-processing one by one, there is many example in Issue. 
Wish you success.

**Comment by pandasayz** [comment]
> Because multi-object penalty coefficient are unbalanced, so we need adjust it, Wang use (NTK or GP) adapty weights method, Deepxde set by parameter loss weights. Model poor performs has multi reason,so we can't make true which important for the problem even this is only a simple problem. The best way is add model-processing one by one, there is many example in Deepxde's Issues. Wish you success.

@xuliang5115  I am a beginner to neural network, you really help me a lot, thank you very much，really,really thank you!!

**Comment by pandasayz** [comment]
@xuliang5115  Sorry to disturb you again, May I ask one more question，could you please tell me if I want to put a small amount of the real value of the equantion to the training progress, how should I  add items in the loss?
